index.d.ts

```diff
@@ -241,6 +242,8 @@ type CommonOptions<IsSync extends boolean = boolean> = {
 
 	This can be an [array of values](https://github.com/sindresorhus/execa#redirect-stdinstdoutstderr-to-multiple-destinations) such as `['inherit', 'pipe']` or `[filePath, 'pipe']`.
 
+	This can also be an [async generator function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function*) to transform the input. Please see the [full documentation here](https://github.com/sindresorhus/execa/tree/main/docs/transform.md).
```

Comment:
```suggestion
	This can also be an async generator function to transform the input. [Learn more.](https://github.com/sindresorhus/execa/tree/main/docs/transform.md)
```

?

Good to keep it short. We can also link to "async generator function" in the full docs.

Reply:
Fixed the other instances in ff1d6785d3d45f302f97d014bb627efc2828febc